assets/chat/js/formatters.js

```diff
@@ -374,6 +374,7 @@ class UrlFormatter {
                 url = self.encodeUrl(m[0]);
                 const extra = self.encodeUrl(decodedUrl.substring(m[0].length));
                 const href = scheme + url;
+                url = href.replace(/(^\w+:|^|www.)|\/\//g, ''); // Removes https://www. from links in chat
```

Comment:
This might be a bit too aggressive, some examples:

| input                                 | expected                      | actual                |
|---------------------------------------|-------------------------------|-----------------------|
| https://www.example.org/www/something | example.org/www/something     | example.org/something |
| https://example.org//some//path       |  example.org//some//path      | example.orgsomepath   |
| https://wwwsitethatstartswithwww.org  |  wwwsitethatstartswithwww.org | itethatstartswithorg  |

Reply:
would this work?
`^(https?:\/\/)?(www\.)?`
https://regex101.com/r/lTfkJb/1

Reply:
SoMuchForSubtlety's seems to work perfect
![VFtxWjs](https://user-images.githubusercontent.com/50147539/60772811-2ae08400-a0c1-11e9-9551-2b86c29afa5e.png)